tunnel.go

```diff
@@ -112,7 +112,7 @@ func setConfigDefaults(config *rest.Config) error {
 	if config.NegotiatedSerializer == nil {
 		s := runtime.NewScheme()
 		c := serializer.NewCodecFactory(s)
-		config.NegotiatedSerializer = serializer.DirectCodecFactory{CodecFactory: c}
+		config.NegotiatedSerializer = serializer.WithoutConversionCodecFactory{CodecFactory: c}
```

Comment:
Is this more deprecations in 1.16?

Reply:
Yes. In 1.15 `DirectCodecFactory` was removed by accident and reintroduced as type alias for `WithoutConversionCodecFactory` with deprecation note, and in 1.16 it was removed.

See at the bottom of https://kubernetes.io/docs/setup/release/notes/#deprecations-and-removals and https://github.com/kubernetes/kubernetes/pull/79263